index.html

```diff
@@ -376,8 +386,13 @@
         scheduleObj.endTime = times[1].trim();
         todaySchedule.push(scheduleObj);
     }
-    if (type == "user") {
-      var scheduleElement = document.querySelector('x-schedule');
+    if (type == "user" || type == "student") {
```

Comment:
This is getting kind of complicated; the schedule object ought to be passed in as a parameter.